test/src/test/java/org/corfudb/runtime/view/ManagementViewTest.java

```diff
@@ -828,16 +837,10 @@ public void testNodeHealing() throws Exception {
         CorfuRuntime corfuRuntime = getRuntime(l).connect();
 
         setAggressiveTimeouts(l, corfuRuntime,
-                getManagementServer(SERVERS.PORT_0).getCorfuRuntime(),
-                getManagementServer(SERVERS.PORT_1).getCorfuRuntime(),
-                getManagementServer(SERVERS.PORT_2).getCorfuRuntime());
-
-        corfuRuntime.getRouter(SERVERS.ENDPOINT_0).getClient(ManagementClient.class)
-                .initiateFailureHandler().get();
-        corfuRuntime.getRouter(SERVERS.ENDPOINT_1).getClient(ManagementClient.class)
-                .initiateFailureHandler().get();
-        corfuRuntime.getRouter(SERVERS.ENDPOINT_2).getClient(ManagementClient.class)
-                .initiateFailureHandler().get();
+                getManagementServer(SERVERS.PORT_0).getManagementAgent().getCorfuRuntime(),
```

Comment:
change this test to first fail and then heal the node.

Reply:
Done.